tests/regression/tests/REQUEST-921-PROTOCOL-ATTACK/921180.yaml

```diff
@@ -17,7 +17,6 @@ tests:
               User-Agent: "OWASP CRS test agent"
               Host: "localhost"
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
-            protocol: "http"
             uri: "/?pineapple=pizza"
```

Comment:
On `921180.yaml` I see lots of `get` requests with the URI still as `/?` and not as `/get?...` is it on purpose?

Reply:
Yes because i was not sure about these. They are missing `method` and are attacks directly on the protocol, so i didn't know if used uri isn't a purpose. What do you think?

Reply:
If you're unsure, I'd leave them be.